Add pointer to GeoCoq (#1)
coq-contribs/tarski-geometry: README.md

> This contribution is no longer maintained, this formalization is now part of the GeoCoq library.
> 
> https://github.com/GeoCoq/GeoCoq

Reviewer: ```suggestion
This formalization is now maintained as part of the GeoCoq library: https://github.com/GeoCoq/GeoCoq
```
@jnarboux everything in this organization is unmainained. I agree that a pointer to GeoCoq is good though.
